File: translations/codenet/uncommented/C++2Go/starcoder/Code/s531853923.go
package main
import "fmt"
func main() {
	var a, b, c, d int
	fmt.Scanf("%d %d %d %d", &a, &b, &c, &d)
	if a*b%2==1 && (c+d)%2==1 {
		fmt.Println("No")
	} else {
		fmt.Println("Yes")
	}
}

File: translations/codenet/uncommented/C++2C/granite/Code/s531853923.c

#include<stdio.h>
int main()
{
	int a, b, c, d;
	scanf("%d %d %d %d", &a, &b, &c, &d);
	if (a*b%2==1&&(c+d)%2==1)
	{
		printf("No\n");
	}
	else
	{
		printf("Yes\n");
	}
}
